lib/Bridge/TolerantParser/WorseReflection/KeywordCompletor.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phpactor\Completion\Bridge\TolerantParser\WorseReflection;
+
+use Generator;
+use Microsoft\PhpParser\Node;
+use Microsoft\PhpParser\Node\ClassMembersNode;
+use Microsoft\PhpParser\Node\Expression\AnonymousFunctionCreationExpression;
+use Microsoft\PhpParser\Node\Expression\MemberAccessExpression;
+use Microsoft\PhpParser\Node\Expression\ScopedPropertyAccessExpression;
+use Microsoft\PhpParser\Node\Expression\Variable;
+use Microsoft\PhpParser\Node\Statement\CompoundStatementNode;
+use Microsoft\PhpParser\Node\StringLiteral;
+use Microsoft\PhpParser\TokenStringMaps;
+use Phpactor\Completion\Bridge\TolerantParser\TolerantCompletor;
+use Phpactor\Completion\Core\Suggestion;
+use Phpactor\TextDocument\ByteOffset;
+use Phpactor\TextDocument\TextDocument;
+use function get_class;
+
+class KeywordCompletor implements TolerantCompletor
+{
+    public const MEMBER_ACCESS = "member_access";
+    public const CLASS_MEMBERS = "class_members";
+    public const STRING_LITERAL = "string_literal";
+    public const VARIABLE = "variable";
+    public const AFTER_ANONYMOUS_FUNC_PARAMS = "after_anonymous_func_params";
+
+    public const SPECIAL_SCOPES = [
+        self::MEMBER_ACCESS => [ ],
+        self::STRING_LITERAL => [ ],
+        self::VARIABLE => [ ],
```

Comment:
can remove these if not used.

Reply:
These are used and say: "In the scope of MEMBER_ACCESS there are no keywords allowed" i.e. don't show keywords there.